Workstream Participant Invitation Policy.md

```diff
@@ -0,0 +1,91 @@
+# WHATWG Workstream Participant Invitation Policy
+
+## Joining the WHATWG via Workstream Participant Invitation
+
+Persons who wish to join the WHATWG and participate in their personal capacity 
+(the <a id="requestor">**Requestor**</a>), but for various reasons are unable to sign the 
+[Contributor and Workstream Participant Agreement] under any of the existing terms
+(for example, the [Individual] is working in the [Field of Web Technologies] and the
+[Entity] employing the individual is unable to join), may request special 
+<a id="ws-invitation">**Workstream Participant Invitation**</a> status. This policy describes the conditions and
+procedures for this process.
+
+The [Steering Group] expects to grant [invitations][Workstream Participant Invitation] sparingly. 
+Wherever possible, persons employed by, contracting, or otherwise working for an
+[Entity] should seek to have the Entity execute the
+[Contributor and Workstream Participant Agreement].
+
+The [Steering Group] must approve an invitation by unanimous consent.
+
+## How to request Workstream Participant Invitation status
+
+1.	File an issue in the [Steering Group's repository](https://github.com/whatwg/sg/issues) requesting
+    [Workstream Participant Invitation] status along with other relevant details or information 
+    relevant to the request.
+2.	The [Steering Group] will discuss the matter with the [Requestor] in the issue.
+3.	Either the [Requestor] or the [Steering Group] may request a private meeting to discuss
+    confidential special considerations.
+4.  The scope of the [Requestor]'s [Workstream] participation is established.
```

Comment:
Can we say something like "The Workstreams the Requestor will participate in are established. or does this mean something else? Perhaps this should also be explicit about the amount of time? That's also listed below, but perhaps that should only cover changes and this should cover the term itself?

Reply:
Sure, happy to re-phrase that to be more clear. I can also mention the 3 year term.